docker-devel.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Shell script for running the devel docker with arguments passed to the script
+#
+# Copyright 2021 Phoenix Systems
+# Author: Hubert Buczynski
+#
+
+DOCKER_IMG_NAME=phoenixrtos/devel
+PATH_TO_PROJECT="$(dirname "$(pwd)/${BASH_SOURCE[0]}")/"
+
+if [ "$#" -eq 1 ] && [ "$1" = "bash" ]; then
```

Comment:
I would go for entering docker (spawning interactive bash) if no parameters are passed and otherwise running desired command

Reply:
Ok, done